doc/nrf/ug_radio_coex.rst

```diff
@@ -0,0 +1,194 @@
+
+Note: When adding custom user implementation a custom dts node can be added, if the ``nrf_radio_coex`` node is unsuitable.
+
+The ``nrf_radio_coex`` dts represents a situation in which a 3 wire interface to the PTA is provided, as shown in the image below. However, the
+role of each of the pins is unspecified and dependent on the Wifi Coexistence implementation.
+
+.. figure:: images/rd_generic_3pin_pta.png
+   :alt: Interfacing of a PTA supported by the ``nrf_radio_coex`` node with nRF52
+
+In order to configure hardware for a PTA using a 3 wire interface:
+
+1. Add the following node in the devicetree file:
+
+   .. code-block::
+
+      / {
+            nrf_radio_coex: radio_coex_three_wire {
+               status = "okay";
+               compatible = "generic-radio-coex-three-wire";
+               req-gpios =     <&gpio0 24 (GPIO_ACTIVE_HIGH)>;
+               pri-dir-gpios = <&gpio0 14 (GPIO_ACTIVE_HIGH)>;
+               grant-gpios =   <&gpio0 25 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
+         };
+      };
+
+#. Optionally replace the node name ``radio_coex_three_wire``.
+#. Replace the pin numbers provided for each of the required properties:
+
+   * ``req-gpios`` - GPIO characteristic of the device that controls the ``REQUEST`` signal of the PTA (request RF medium access).
+   * ``pri-dir-gpios`` - GPIO characteristic of the device that controls the ``PRIORITY`` signal of the PTA (priority + RX/TX direction).
+   * ``grant-gpios`` - GPIO characteristic of the device that controls the ``GRANT`` signal of the PTA (RF medium access granted).
+
+   Type ``phandle-array`` is used here, which is common in Zephyr's devicetree to describe GPIO signals.
+   The first element ``&gpio0`` refers to the GPIO port ("port 0" has been selected in the example shown).
+   The second element is the pin number on that port.
+
+#. On nRF53 devices, you must also apply the same devicetree node mentioned in step 2 to the network core.
+   To do so, apply the overlay to the correct network core child image by creating an overlay file named :file:`child_image/*childImageName*.overlay` in your application directory, for example :file:`child_image/multiprotocol_rpmsg.overlay`.
+
+   The ``*childImageName*`` string must be one of the following values:
+
+   *  ``multiprotocol_rpmsg`` for multiprotocol applications having support for both 802.15.4 and Bluetooth.
+   *  ``802154_rpmsg`` for applications having support for 802.15.4, but not for Bluetooth.
+   *  ``hci_rpmsg`` for application having support for Bluetooth, but not for 802.15.4.
+
+
+
+WiFi Coexistence implementations
+********************************
+
+The following CX implementations are available:
+
+* :ref:`Generic 3-pin interface`
+* :ref:`"Thread" implementaion`
+
+Each implementaion is a plugin consisting of one or more C source files interfacing with the MPSL API (which in turn communicates with the
+802.15.4 Radio Driver)
```

Comment:
The rest of the document is more generic and refers to "SR". Even though MPSL CX is not supported with BLE yet, I think we should be consistent and not refer directly to the 802.15.4 RD here.

Reply:
You are right - fixed this and also changed the name of the picture rd_generic3pin_pta.png to coex_generic3pin_pta.png